CMakeLists.txt

```diff
@@ -363,7 +407,7 @@ target_include_directories(fc
     ${OPENSSL_INCLUDE_DIR}
     "vendor/diff-match-patch-cpp-stl"
     ${CMAKE_CURRENT_SOURCE_DIR}/vendor/websocketpp
-    "${editline_includes}"
+    ${CMAKE_CURRENT_SOURCE_DIR}/vendor/editline
```

Comment:
I believe this line is superfluous, because vendor/editline doesn't contain any includes, and vendor/editline/include should have been taken care of by setting INTERFACE_INCLUDE_DIRECTORIES above.